src/components/SignIn/SignIn.js

```diff
@@ -13,6 +13,7 @@ const SignIn = ({ actions, auth }) => {
 
   return (
     <div>
+      <h2>GoPlan</h2>
       {auth.isAuthenticated ? (
         <button onClick={actions.auth.signOut}>Sign Out</button>
```

Comment:
I don't think we need this anymore right?

Reply:
I removed the HomePage file, so theres no title right now on the login page.

Reply:
I mean the sign out button and `auth.isAuthenticated ?`